conda/petsc/build.sh

```diff
@@ -56,15 +39,17 @@ configure_petsc \
     FFLAGS="$FFLAGS" \
     FCFLAGS="$FCFLAGS" \
```

Comment:
Actually, looks like we used to say FCFLAGS = FFLAGS to begin with and then modify further. Would you like to set this line to `FCFLAGS="$FFLAGS"` instead of removing it?